python/tank_vendor/shotgun_authentication/session_cache.py

```diff
@@ -29,9 +29,11 @@
 from tank_vendor.shotgun_api3.lib import httplib2
 from tank_vendor import yaml
 from .errors import AuthenticationError
-import logging
+from .sg_auth_logging import get_logger
 
-logger = logging.getLogger("sg_auth.session")
+from ..shotgun_base import get_cache_root, get_site_cache_root
+
+logger = get_logger("session")
```

Comment:
I'm curious about the removal of the prefixed logger names. I realize they weren't consistent, but wondering why no prefix anymore. Perhaps a client might want to filter all shotgun/toolkit related logging?